contracts/Org.sol

```diff
@@ -47,6 +47,7 @@ contract Org is Administratable {
     public
     onlyAdminOrRole(adminContractAddress, IEndaomentAdmin.Role.ORG_FACTORY)
   {
```

Comment:
Can we add a special check for if the ein is 0 and test this case.

Reply:
If the ein is 0 it will fail the require statement of `require(ein >= 10000000 && ein <= 999999999`, so we don't need a special check for that

Reply:
Okay sg.